packages/strapi-admin/index.js

```diff
@@ -9,6 +10,64 @@ const chokidar = require('chokidar');
 
 const getPkgPath = name => path.dirname(require.resolve(`${name}/package.json`));
 
+function getCustomWebpackConfig(dir, config) {
+  const adminConfigPath = path.join(dir, 'admin', 'admin.config.js');
+  let webpackConfig = _.cloneDeep(getWebpackConfig(config));
```

Comment:
deep cloning the webpackconfig might create unintended bugs as it is not just a plain js object but also contains some plugin instance an such.

Reply:
So you suggest I don't clone it letting the user mutate it directly?

Reply:
yes I think it is why other are doing it like this.